core/src/cs/eng1/piazzapanic/food/ingredients/BasicGrillable.java

```diff
@@ -20,8 +21,9 @@ public BasicGrillable(String type, FoodTextureManager textureManager) {
     @Override
     public boolean grillTick(float delta) {
         accumulator += delta;
-
-        if (accumulator >= grillStepTime) {
+        if (accumulator >= grillStepTime + failTime) {
+            setUseable(false);
```

Comment:
Why does `BasicCookable` and `BasicGrillable` set `usable` in different ways?

Reply:
I implemented a setUseable function when doing BasicGrillable but forgot to bring it into the other methods, will update to fix those.